src/main/java/com/twilio/oai/TwilioNodeGenerator.java

```diff
@@ -48,7 +48,6 @@ public void processOpts() {
         additionalProperties.put("domain", StringUtils.camelize(domain));
 
         supportingFiles.clear();
-        supportingFiles.add(new SupportingFile("tsconfig.mustache", "tsconfig.json"));
```

Comment:
Nit: Any compilation impact on deleting tsconfig.json?

Reply:
Nope. We should have one defined at the repo-root so these aren't necessary.